pa/src/components/CreateButton.vue

```diff
@@ -34,11 +38,24 @@
         },
         methods: {
             getCreationOptions: function () {
-                BASE
+                this.fin_api
                     .options(this.API_URL)
                     .then(response => {
                         if (response.data) {
                             let creationData = response.data['actions']['POST'];
+
+                            if (creationData['query_params']) {
+                                let query_params = creationData['query_params'];
+                                delete creationData['query_params'];
+                                for (const query_param in query_params) {
```

Comment:
use `let` instead of `const` and `of` instead of `in`.
Read the difference, we fucked up security once using in instead of `of`